CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### 0.6.0+1
+
+* Updated use of deprected Dart core constants.
+
+### 0.6.0
```

Comment:
@pq – looks like 0.6.0 was never shipped - https://pub.dartlang.org/packages/dart_coveralls

Sorry! Can go back to 0.6.0 – prepare for release.